app/src/androidTest/java/com/android/partagix/stamp/StampTest.kt

```diff
@@ -0,0 +1,111 @@
+package com.android.partagix.stamp
+
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.android.partagix.model.StampViewModel
+import com.android.partagix.screens.StampScreen
+import com.android.partagix.ui.navigation.NavigationActions
+import com.android.partagix.ui.navigation.Route
+import com.android.partagix.ui.screens.StampScreen
+import com.kaspersky.components.composesupport.config.withComposeSupport
+import com.kaspersky.kaspresso.kaspresso.Kaspresso
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import io.github.kakaocup.compose.node.element.ComposeScreen.Companion.onComposeScreen
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.impl.annotations.RelaxedMockK
+import io.mockk.just
+import io.mockk.mockk
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class StampTest : TestCase(kaspressoBuilder = Kaspresso.Builder.withComposeSupport()) {
+  @get:Rule val composeTestRule = createComposeRule()
+  @RelaxedMockK lateinit var mockNavActions: NavigationActions
+  @RelaxedMockK lateinit var mockStampViewModel: StampViewModel
+
+  @Before
+  fun testSetup() {
+
+    mockStampViewModel = mockk()
+
+    mockNavActions = mockk<NavigationActions>()
+    every { mockNavActions.navigateTo(Route.HOME) } just Runs
```

Comment:
```suggestion
```
You can't delete alle the navigateTo (keep the go back because you use it) but the rest are useless as you do not navigate between the screens